### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by JulianKniephoff**
See individual commits for details.

### CODE REVIEW

File: backend/src/main.rs [hunk 1]
```diff
@@ -35,7 +38,23 @@ async fn main() -> Result<()> {
...
+                build_info::GIT_COMMIT_HASH.unwrap(),
+                if let Some(true) = build_info::GIT_DIRTY {
+                    ", dirty"
+                } else {
+                    ""
+                },
+                build_info::BUILT_TIME_UTC,
+            ))
+            .get_matches(),
+    );
```
Comment (LukasKalbertodt) [inline backend/src/main.rs:46]: I would prefer to extract the `format!()` thingy and create a new variable `version`. This avoids this very complex expression and improves readability IMO.

EDIT: ah, I see, you change it next commit. This commit could have had the `version` function already but oh well, doesn't matter I guess :D

File: backend/src/main.rs [hunk 2]
```diff
@@ -117,3 +106,20 @@ fn load_config_and_init_logger(args: &args::Shared) -> Result<Config> {
 
     Ok(config)
 }
+
+pub fn version() -> String {
+    mod build_info {
+        include!(concat!(env!("OUT_DIR"), "/built.rs"));
+    }
```
Comment (LukasKalbertodt) [inline backend/src/main.rs:113]: Huff, I didn't know that's possible! Do we really want that? Is the module visible outside of this function? So many questions. Intuitively I would say "please don't declare mod in function"...

File: backend/src/main.rs [hunk 3]
```diff
@@ -22,7 +22,6 @@ mod sync;
 mod util;
 
 mod build_info {
-    include!(concat!(env!("OUT_DIR"), "/built.rs"));
 }
```
Comment (LukasKalbertodt) [inline backend/src/main.rs:25]: Even weirder, here you forgot to remove the `mod` thingy so there is another module here?! What :D How are even semantics?!

File: backend/src/main.rs [hunk 4]
```diff
@@ -117,3 +106,20 @@ fn load_config_and_init_logger(args: &args::Shared) -> Result<Config> {
 
     Ok(config)
 }
+
+pub fn version() -> String {
```
Comment (LukasKalbertodt) [inline backend/src/main.rs:110]: Docs? :D Just one line maybe?

File: frontend/src/i18n/locales/de.yaml [hunk 5]
```diff
@@ -11,6 +11,7 @@ navigation: Navigation
 rename: Umbenennen
 this-field-is-required: Dieses Feld ist erforderlich.
 copy-to-clipboard: In Zwischenablage kopieren
+version-information: Versionsauskunft
```
Comment (LukasKalbertodt) [inline frontend/src/i18n/locales/de.yaml:14]: Uff, that sounds verry german to me :D How about just "Version"? Or even, "Versionsinformationen" as a more direct translation.

**Review by LukasKalbertodt — CHANGES_REQUESTED** [review]
Sweet. Only some comments about smaller stuff.

File: backend/src/main.rs (hunk 1 above)
Comment (JulianKniephoff) [reply backend/src/main.rs]: I got you :ok_hand:

File: frontend/src/i18n/locales/de.yaml (hunk 5 above)
Comment (JulianKniephoff) [reply frontend/src/i18n/locales/de.yaml:14]: I mean it **is** the German translation after all. :stuck_out_tongue_winking_eye: But sure, we can just have it say "version". :shrug:

File: backend/src/main.rs (hunk 2 above)
Comment (JulianKniephoff) [reply backend/src/main.rs:114]: It doesn't seem visible, so it's fine, I guess? :shrug:

File: backend/src/main.rs (hunk 3 above)
Comment (JulianKniephoff) [reply backend/src/main.rs]: Whoops :sweat_smile:

**Review by LukasKalbertodt — APPROVED** [review]

**Pull request merged**